stubs/invoke/METADATA.toml

```diff
@@ -1,5 +1,5 @@
 version = "2.0.*"
-obsolete_since = "2.1.0" # Released on 2023-04-29
+obsolete_since = "2.1.2" # Released on 2023-05-16
```

Comment:
According to https://pypi.org/project/invoke, it was released on 2023-05-15. Stubsabot agrees in #10098 (it's auto-updated the PR to suggest marking the stubs as obsolete, now that the py.typed file is included in the wheel uploaded to PyPI)

Reply:
timezone difference :)